### ISSUE TITLE
local images load failed

### ISSUE THREAD

**Issue by tylvn** [Saul-Mirone/milkdown-vscode#25]
For example
```
![](image/微生物学笔记/1641386328583.png)

![](image/微生物学笔记/1641386612027.png)
```
Above is grammar about local images in markdown. In [marktext](https://github.com/marktext/marktext) and vscode, load image success. However, it is fail to load images in milkdown.

VS Code

<img width="721" alt="image" src="https://user-images.githubusercontent.com/95011132/159653316-fcf190ce-f788-4d27-8e87-1606f34aaf5a.png">

Milkdown

<img width="760" alt="image" src="https://user-images.githubusercontent.com/95011132/159653196-6d84be9b-5af1-4e1a-81c8-d68a70dc40a8.png">

**Comment by rchangl** [comment]
Same here。

**Comment by rchangl** [comment]
Same here

**Comment by Saul-Mirone** [comment]
Resolved by https://github.com/Saul-Mirone/milkdown-vscode/commit/577e5734fba9ca3616452d3fd0adfdb7a9896ce4

**Comment by Saul-Mirone** [comment]
Released with 0.0.13.

**Comment by tylvn** [comment]
Thanks for help, I will try it if I have time.👍
